CenterForOpenScience/osf.io
# [Feature] Populate rename text in Fangorn [OSF-7151]

## Purpose
Auto populating name on fangorn rename was removed due to various bugs.


## Changes
Added populating back in without the bugs.


## Side effects
None known.



## Ticket
https://openscience.atlassian.net/browse/OSF-7151


[#OSF-7151]

## QA Considerations
1. Check that all fangorn inputs work properly (filter, rename, folder create)
2. Check that renaming works
3. Check that there were no regressions (] and " ' all work, etc)
4. Also check for regressions of the last letter disappearing bug on mouse out
